internal/cli/docker_prune.go

```diff
@@ -14,6 +17,18 @@ type dockerPruneCmd struct {
 	maxAgeStr string
 }
 
+type dpDeps struct {
```

Comment:
This type and the constructor aren't used?

Reply:
lol ya they belong in a different PR, just removed them